service/README.md

```diff
@@ -0,0 +1,40 @@
+# The FuzzBench Service
```

Comment:
Maybe this README should be in the normal place for docs (I can create a fuzzbench maintainers section to put it in)?

Reply:
Yes should be in main docs.